site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "High Availability Agones"
+date: 2023-02-08T15:02:23Z
+weight: 20
+description: >
+  Allows controllers to be strictly for control loops; feature separates extensions and webhooks out of your controllers
+---
+
+## High Availability for Agones Controller
+
+The `agones-controller` pod is currently not only used for control loops of game server related resources, but also include other services such as webhooks and allocation extension srvices. High availability agones mean as opposed to having controller host everything mentioned, we will add additional `extensions` pods that exposes the webhook and allocations service. Thus, making `agones-controller` pod strictly for control loops. Read more about pod disruption [here](https://agones.dev/site/docs/advanced/controlling-disruption/)
+
+{{< alpha title="`split controller and extensions` API" gate="SplitControllerAndExtensions" >}}
+
+{{< alert title="GKE Autopilot Clusters" color="warning" >}}
+In order to use GKE Autopilot clusters with Agones, this feature flag `SplitControllerAndExtensions` must be enabled along with `SafeToEvict` (should already be enabled). 
```

Comment:
Make this warning more concise:

---

`SplitControllerAndExtensions` must be enabled on GKE Autopilot.

---

Alternately drop this and I'll doc it with Autopilot stuff later - ideally this should link off to Autopilot docs we don't have yet.